test/e2e/app-dir/next-after-app-deploy/index.test.ts

```diff
@@ -0,0 +1,116 @@
+/* eslint-env jest */
+import { nextTestSetup, isNextDev, isNextDeploy } from 'e2e-utils'
+import { retry } from 'next-test-utils'
+
+const runtimes = ['nodejs', 'edge']
+
+const WAIT_BEFORE_REVALIDATING = isNextDeploy ? 10_000 : 5_000
```

Comment:
need to check what value is appropriate for a deploy test -- ideally, we'd like it to wait long enough that the invocation would die if the callback was scheduled with `setTimeout` instead of `unstable_after`

Reply:
okay i got confused and forgot that our `waitUntil` can't keep anything alive past `maxDuration`

> Promises passed to waitUntil() will have the same timeout as the function itself. If the function times out, the promises will be cancelled.
https://vercel.com/docs/functions/functions-api-reference#waituntil

so this doesn't really matter.